bin/pycbc_compress_bank

```diff
@@ -125,8 +129,8 @@ else:
 logging.info("getting needed dfs")
 # we'll ensure that the there are atleast 2 samples in a waveform
 seg_lens = 4*numpy.array([max(4./args.sample_rate,
-    2**numpy.ceil(numpy.log2(compress.rough_time_estimate(m1, m2, fmin))))
-    for m1,m2 in zip(templates.mass1, templates.mass2)])
+    2**numpy.ceil(numpy.log2(compress.rough_time_estimate(m1, m2, flow))))
+    for m1,m2,flow in zip(templates.mass1, templates.mass2, templates.f_lower)])
```

Comment:
@soumide1102 : Ok, @ahnitz just informed me that when you initialize `FilterBank` with `low_frequency_cutoff=opts.low_frequency_cutoff`, it will override whatever is in the bank with the provided value. So that means, no, you don't have to change anything here. However, that also means you don't need to have if/elses in the loop below...